ui/src/app/modules/domain/column-response.ts

```diff
@@ -15,41 +15,41 @@
  * limitations under the License.
  */
 
-import { Item, ItemSorter } from './column/item-sorter';
+import { ActionItem } from './action-item';
 import { Thought } from './thought';
 
+export type ColumnItem = Thought | ActionItem;
+
+export type ColumnItems = ColumnItem[];
+
 export interface ColumnResponse {
   id: number;
-  items: ItemSorter;
+  items: ColumnItems;
   title: string;
   topic: string;
 }
 
 export function emptyColumnResponse(): ColumnResponse {
   return {
     id: -1,
-    items: { active: [], completed: [] },
+    items: [],
     topic: '',
     title: '',
   };
 }
 
-export function removeItemFromColumn(response: Item, items: ItemSorter): void {
-  removeItemFromList(items.active, response);
-  removeItemFromList(items.completed, response);
+export function removeItemFromColumn(columnItem: ColumnItem, items: ColumnItems): void {
```

Comment:
We could probably move the `removeItemFromList` call to the top and not have this function call intermediate.